server/bin/unittests

```diff
@@ -678,6 +700,19 @@ declare -A cmds=(
     # Tests that the behavior of the find command works as we expect to
     # ensure that buckets for pbench-unpack-tarballs work properly.
     [test-25]="_run test-find-behavior"
+
+    # Test to Log messages on File
+    [test-26.1]="_run_pytest test_logger_type"
```

Comment:
Pytest should not need external server configurations to get different behaviors.  With Pytest we can provide all the inputs the unit test needs.  Can we encode all this in the test python code?

Reply:
Isn't the point of the test to specify the logging method *in the config file* and then make sure that the message ends up in the expected place?

Reply:
Yes, but you can make the config file object in Pytest.  The config file parsing and all that is secondary.